configure

```diff
@@ -25,12 +25,20 @@ check_util make
 check_util gcc
 check_util lsblk
 
-if [ ! -e /lib/modules/$(uname -r)/build/ &> /dev/null ]
+`rpm -qa | grep kernel-head &> /dev/null`
```

Comment:
rpm is not installed by default on every os (i.e. ubuntu)

Reply:
Great thanks to You.